### ISSUE TITLE
Use plottly for rendering of graphs

### ISSUE THREAD

**Issue by JonasCir** [healthIMIS/ci-analyzer#2]
I used plottly.js in the past and its super simple to use and gives beautiful, interactive graphs.

**Comment by DavidBaldsiefen** [comment]
If it's easier to use, go ahead :) But please don't do large changes to DataFetcher.js (yet), as I want to clean it up a little

**Comment by JonasCir** [comment]
Yeah it's pretty simple, if you clean up, please split the part which fetches data and the part which renders the data. by this it's easier to introduce it. Sure I'll wait some time...

**Comment by DavidBaldsiefen** [comment]
> 
> 
> Yeah it's pretty simple, if you clean up, please split the part which fetches data and the part which renders the data. by this it's easier to introduce it. Sure I'll wait some time...

It's already split up, in the` index.html `is a `drawResults(timings)` function which is called whenever all the data is fetched. Timings is an array containing each time in Minutes. If the value `smoothness `is > 1, this array already containes the smoothed values. In theory I would prefer to have all the JavaScript code outside of the index.html but that's just a pet-peeve of mine ;)

**Comment by DavidBaldsiefen** [comment]
Now that I think about it, I might actually change the array which is returned to contain more infos, such das dates and links to the relevant jobs. But for now, just assume `timings `contains times like this: `[12.43, 14.32, 10.0]`
